arangod/VocBase/Methods/Collections.cpp

```diff
@@ -1289,19 +1289,31 @@ arangodb::Result Collections::checksum(LogicalCollection& collection,
   return trx.finish(res);
 }
 
+/// @brief the list of collection attributes that are allowed by user-input
+/// this is to avoid retyping the same list twice
+#define COMMON_ALLOWED_COLLECTION_INPUT_ATTRIBUTES                            \
+  StaticStrings::DataSourceSystem, StaticStrings::DataSourceId, "keyOptions", \
+      StaticStrings::WaitForSyncString, StaticStrings::CacheEnabled,          \
+      StaticStrings::ShardKeys, StaticStrings::NumberOfShards,                \
+      StaticStrings::DistributeShardsLike, "avoidServers",                    \
+      StaticStrings::IsSmart, StaticStrings::ShardingStrategy,                \
+      StaticStrings::GraphSmartGraphAttribute, StaticStrings::Schema,         \
+      StaticStrings::SmartJoinAttribute, StaticStrings::ReplicationFactor,    \
+      StaticStrings::MinReplicationFactor, /* deprecated */                   \
+      StaticStrings::WriteConcern, "servers"
```

Comment:
"servers" as StaticString?

Reply:
Static Strings are good point, but as this shall be a bug-fix I would do this in a different refactoring PR.